src/gui/editors/AutomationEditor.cpp

```diff
@@ -2102,5 +2331,37 @@ void AutomationEditorWindow::updateWindowTitle()
 	setWindowTitle( tr( "Automation Editor - %1" ).arg( m_editor->m_clip->name() ) );
 }
 
+/**
+ * @brief This method handles the AutomationEditorWindow event of changing
+ * progression types. After that, it calls updateEditTanButton so the edit
+ * tangents button is updated accordingly
+ * @param Int New progression type
+ */
```

Comment:
I've mostly seen those being written in the .cpp files, but a quick search is showing them on both headers and source files. Should I still change it or leave it for when we move every method doc to headers?

Reply:
> I've mostly seen those being written in the .cpp files, but a quick search is showing them on both headers and source files. Should I still change it or leave it for when we move every method doc to headers?

I would move any documentation you have into the headers. We mustn't move all the documentation into headers at once in a separate PR. It can always happen gradually.

Reply:
Cool, I moved the documentation added on this PR to the headers then!